core/src/test/scala/io/snappydata/CommandLineToolsSuite.scala

```diff
@@ -273,4 +273,10 @@ class CommandLineToolsSuite extends SnappyTestRunner {
       ("rm -r ./SNAP-2631-work-lead").!!
     }
   }
+  test("test_run_command"){
+    val (out, err) = executeCommand(snappyProductDir +
+        "/bin/snappy run file='somefile.sql'")
+    assert(!out.contains("[-locators=<addresses>]"),
+      s"-locators options has been removed successfully")
```

Comment:
The message in assert() is for failure case, it is displayed when the assert statement fails. So it should be like "-locators option still displayed in run command's usage text!"

Reply:
Modified as per comment